config/admin.txt

```diff
@@ -1,75 +0,0 @@
-# This is an example config file.
```

Comment:
This file shouldn't be deleted.

Reply:
It's an admin.txt file I've made to get myself admin stuff.